web/src/settings_bots.js

```diff
@@ -418,7 +418,9 @@ export function show_edit_bot_info_modal(user_id, from_user_info_popover) {
         $("#edit_bot_modal .dialog_submit_button").prop("disabled", true);
 
         function get_options() {
-            const user_ids = people.get_active_human_ids();
+            const user_ids = people.get_realm_active_human_user_ids();
+            // Remove current owner.
+            user_ids.splice(user_ids.indexOf(owner_id), 1);
```

Comment:
I guess this was added by mistake during solving merge conflicts.

Reply:
Yeah, it's because of the merge. I will look into it and correct it.